nx-X11/programs/Xserver/xkb/Imakefile

```diff
@@ -40,7 +40,7 @@ XKB_DDXDEFS = XkbServerDefines
    LINTLIBS = ../dix/llib-ldix.ln ../os/llib-los.ln
 
 DEFINES = $(XKB_DDXDEFS) $(NX_DEFINES)
-XKB_DEFINES = -DXKB_BASE_DIRECTORY=\"/usr/share/X11/xkb\" -DXKB_BIN_DIRECTORY=\"/usr/bin\" -DXKB_DFLT_RULES_FILE=\"base\" -DXKB_DFLT_KB_LAYOUT=\"us\" -DXKB_DFLT_KB_MODEL=\"pc102\" $(XKB_DISABLE)
+XKB_DEFINES = -DXKB_BASE_DIRECTORY=\"$(SYSTEMXKBCONFDIR)\" -DXKB_BIN_DIRECTORY=\"$(SYSTEMXKBBINDIR)\" -DXKB_DFLT_RULES_FILE=\"base\" -DXKB_DFLT_KB_LAYOUT=\"us\" -DXKB_DFLT_KB_MODEL=\"pc102\" $(XKB_DISABLE)
```

Comment:
ah, here we go... are the XKB RLMVO defaults sooo default that we should hard code them here? (Repeating myself...)

Reply:
well, we need some default, us and pc102  are the values nx has always used, base is the best choice nowadays (and is identical to xfree86 that nx was hsipped with). We can of course change them to something else, but I don't see much benefit, since x2go (and with bad luck also the desktop) will set it anyway. IMHO there cannot be an automatism at the compile time, this is client side runtime decision. As I stated in the pr message I want to rebase my long-time pending Keyboard overhaul once this pr is merged. Then I can easily add the runtime RMLVO auto-detection.

Reply:
ok, then please fix commit msg of 25ca9b0 and we are good to go.